environments/key4hep-common/packages.yaml

```diff
@@ -76,13 +84,12 @@ packages:
     - spec: openssh@99
       prefix: /usr
 
-
   root:
-    require: +davix+fftw+gsl+math+minuit+mlp+opengl~postgres~pythia6+pythia8+python+r+root7+roofit+rpath~shadow+sqlite+ssl+tbb+threads+tmva+unuran+vc+vdt+x+xml+xrootd cxxstd=20
+    require: '@6.30: +davix+fftw+gsl+math+minuit+mlp+opengl~postgres~pythia6+pythia8+python+r+root7+roofit+rpath~shadow+sqlite+ssl+tbb+threads+tmva+unuran+vc+vdt+x+xml+xrootd cxxstd=20'
```

Comment:
Did you ever dig deeper in what is pulling in `6.28` if not asking for `6.30`?

Reply:
Nope :cry: 
I also see ROOT rebuilding after reconcretizing with the same settings for some reason...

Reply:
Yeah, same for me. In my case it could also be a changed dependency, but I haven't investigated any further yet.

Reply:
Apparently it seems cxxstd of xrootd can be either 11 and 14 and sometimes it's one or the other even though nothing has changed. I'll try to hardcode it and see if it helps... It seems only to happen in alma9